ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/models.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+import json
+from functools import cached_property
+
+from ansible.errors import AnsibleError
+from anta.models import AntaTestCommand
+
+
+class InventoryDeviceHttpApi:
+    """
+    Custom implementation of an ANTA InventoryDevice with Ansible HttpApi connection session object.
+    """
+
+    def __init__(self, session):
```

Comment:
Maybe add a type hint on the session object and maybe even do a `isinstance` check since we expect it to have certain methods below.